app/controllers/admin/conference_controller.rb

```diff
@@ -89,7 +87,7 @@ def update
         redirect_to edit_admin_conference_path(id: @conference.short_title),
                     notice: 'Conference was successfully updated.'
       else
-        redirect_to edit_admin_conference_path(id: short_title),
+        redirect_to edit_admin_conference_path(id: @conference.short_title),
```

Comment:
If the user tries to update the conference and the short_title is left blank, this line is gonna crash.